release/downloadIstioCandidate.sh

```diff
@@ -36,10 +36,12 @@ else
   OSEXT="linux"
 fi
 
+# Determines the istioctl version.
```

Comment:
minor nit: Message should be Istio version. And more specifically, the latest Istio version in version sorting order.

So maybe: # Determine the latest Istio version by version number ignoring alpha, beta, and rc versions.

Reply:
Updated in latest commit.